conda_forge_tick/make_migrators.py

```diff
@@ -797,7 +798,11 @@ def initialize_migrators(
     gx: nx.DiGraph,
     dry_run: bool = False,
 ) -> MutableSequence[Migrator]:
-    migrators: List[Migrator] = []
+    migrators: List[Migrator] = [
```

Comment:
We should append this migrator to the list after the noarch python migrator. There are some conventions in the code about where each kind of migrator goes in the list.